docs/src/pages/components/tooltips/tooltips.md

```diff
@@ -92,21 +92,6 @@ By default disabled elements like `<button>` do not trigger user interactions so
 
 {{"demo": "pages/components/tooltips/DisabledTooltips.js"}}
 
-> If you're not wrapping a Material-UI component that inherits from `ButtonBase`, for instance, a native `<button>` element, you should also add the CSS property _pointer-events: none;_ to your element when disabled:
-
-```jsx
-<Tooltip title="You don't have permission to do this">
-  <span>
-    <button
-      disabled={disabled}
-      style={disabled ? { pointerEvents: 'none' } : {}}
-    >
-      A disabled button
-    </button>
-  </span>
-</Tooltip>
-```
-
```

Comment:
This is still true, why remove it?
https://codesandbox.io/s/simpletooltips-material-demo-forked-s1pqe?file=/demo.tsx